packages/core/content-type-builder/admin/src/components/FormModalNavigationProvider/index.js

```diff
@@ -8,6 +8,23 @@ const FormModalNavigationProvider = ({ children }) => {
   const [state, setFormModalNavigationState] = useState(INITIAL_STATE_DATA);
   const { trackUsage } = useTracking();
 
+  const onClickSelectCustomField = ({ attributeType, customFieldUid }) => {
+    if (state.forTarget === 'contentType') {
+      trackUsage('didSelectContentTypeFieldType', { type: attributeType });
```

Comment:
We'll probably want to track custom fields usage in particular. We could ask Emilie or Aurélien

Reply:
Ok I will ask Emilie but I think we will probably handle that in another PR